samples/nodejs/README.md

```diff
@@ -17,6 +17,8 @@ You must have `libindy.dylib` at `/usr/local/lib/libindy.dylib` before running n
 ### On Windows
 [Install Libindy](https://github.com/hyperledger/indy-sdk#windows)
 
+Set environment variable `LD_LIBRARY_PATH` to include a path to the lib folder that contains indy.dll.lib
+
```

Comment:
should this be `contains indy.dll` ?

Reply:
It must contain both `indy.dll.lib` AND `indy.dll`

Reply:
...Which the Windows builds include...

Reply:
I mentioned `indy.dll.lib` specifically because this is the file that is explicitly included by node-gyp as it has the static method definitions